resources/css/utilities.css

```diff
@@ -1,3 +1,7 @@
+[x-cloak] {
```

Comment:
`x-cloak` is currently not configured correctly in the project.

ref: https://alpinejs.dev/directives/cloak